.github/workflows/stale.yml

```diff
@@ -11,15 +11,17 @@ jobs:
 
     if: github.repository == 'optuna/optuna'
     steps:
-    - uses: actions/stale@v3
+    - uses: actions/stale@v4
```

Comment:
Just a curiosity, but the latest version of stale bot seems to be [v5.0.0](https://github.com/actions/stale/releases/tag/v5.0.0). Do we have any reason to use `v4`? I guess the options used in this workflows are still listed in the [input option tables](https://github.com/actions/stale/blob/65d24b70926a596b0f0098d7e1eb572175d73bc1/README.md?plain=1#L27).

Reply:
Actually, I've just followed the example here: https://github.com/actions/stale#usage
We can just use the latest version, though. I'll update the PR.